File: Day 24/Aayushi-Mittal.cpp
//To print all the unique elements in an array.

#include <bits/stdc++.h>
using namespace std;
int main()
{
  int n, arr[n];
  cin>>n;
  for(int i=0; i<n; i++)
    cin>>arr[i];
  sort(arr, arr+n);
  cout<<"Unique elements in the array: ";
  for(int i=0; i<n; i++)
  {
    if(arr[i+1]!=arr[i] && i+1<n)
      cout<<arr[i]<<", ";
  }
}

File: Day 24/Aayushi-Mittal.py
#To print all the unique elements in an array.

n=int(input()) 
arr=[] 
for i in range(0,n):
    arr.append(int(input()))
arr.sort()
print("Unique Elements of the array are:", end=" ")
for i in range(0,n):
    if i+1<n and arr[i+1]!=arr[i]:
        print(arr[i], end=", ")
